### PR TITLE
[Bahasa Indonesia] - ID Translation of /web/billions/

### PR DESCRIPTION
**Pull request by abdshomad**
Hi @petele, 
CC: @tyohan, @mychaelgo,

I have finished my next Indonesian Translation of Web - Building for Billions /web/billions/ section. 

I have tested it running on my local development version. Here's the screenshot:
web-billions

Please review it.

### CODE REVIEW

File: src/content/id/_redirects.yaml [hunk 1]
```diff
@@ -0,0 +1,338 @@
+redirects:
```
Comment (petele) [inline src/content/id/_redirects.yaml]: This file should not be included - we only need it in the English directory

File: src/content/id/_upper_tabs-Fundamentals.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/_upper_tabs-Fundamentals.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_book.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/_book.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_common-links.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{% comment %}
```
Comment (petele) [inline src/content/id/_common-links.md]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_project.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/_project.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_upper_tabs-Showcase.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/_upper_tabs-Showcase.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_upper_tabs-Tools.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/_upper_tabs-Tools.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_upper_tabs-Updates.yaml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/_upper_tabs-Updates.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/fundamentals/_book.yaml [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/fundamentals/_book.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/fundamentals/_redirects.yaml [hunk 10]
```diff
@@ -0,0 +1,137 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/fundamentals/_redirects.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/fundamentals/getting-started/_redirects.yaml [hunk 11]
```diff
@@ -0,0 +1,5 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/fundamentals/getting-started/_redirects.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/fundamentals/getting-started/_toc.yaml [hunk 12]
```diff
@@ -0,0 +1,37 @@
+toc:
```
Comment (petele) [inline src/content/id/fundamentals/getting-started/_toc.yaml]: Please remove this file - it shouldn't be included in the localized folders.

**Comment by petele** [comment]
@abdshomad - it also looks like this contains the same changes that are in #3694 and #3696

**Comment by abdshomad** [comment]
@petele - Included files are deleted. Please re-check.

**Comment by petele** [comment]
Looks like it still has a few extra files in it, can you remove those please?

**Comment by abdshomad** [comment]
Hi @petele, 

Remainings is: src/content/id/_footer.yaml
and I was intended to translate this file. Since I assume that this include file also should not be translated, I remove it from this PR. 

Please re-check it. Thank you :)

**Pull request merged**